Chore / update favicons

sbrody opened the pull request:
Updates set of favicons as per govuk-frontend 5.1 https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#include-the-new-logo-assets

RobjS reviewed (changes requested):
I think the more robust fix for this would be to undo the change Serena made here: https://github.com/dxw/govuk-blogs/commit/4f1051215e36886f751295c3553014696c3ace07#diff-02e3066b5b6f061ff00ba30404c5867a3bbb4db774eaea673c6344fda5917af0 (as a temporary workaround to get the new crown logos working) and go back to referencing the files in the `build/` folder, which are automatically copied over from the node package as part of the Grunt build. Then if these change again in future, we should just have to re-run Grunt build to pick up the new versions, without having to manually copy them across.

RobjS reviewed (changes requested):
@sbrody Sorry, I'm looking at this again in more detail and realising that a lot of the images referenced in the head don't actually seem to exist in the govuk-frontend assets. e.g. I'm getting 404s for:

http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-152x152.png
http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon.png
http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-mask-icon.svg
http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-180x180.png
http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-167x167.png

I'm not sure where Serena got those images from (maybe she generated them herself?) but this might be a good opportunity to bring our markup inline with what the govuk-frontend suggests here: https://design-system.service.gov.uk/styles/page-template/, so that the images referenced are the ones that exist in the govuk-frontend assets.

sbrody commented:
> @sbrody Sorry, I'm looking at this again in more detail and realising that a lot of the images referenced in the head don't actually seem to exist in the govuk-frontend assets. e.g. I'm getting 404s for:
> 
> http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-152x152.png http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon.png http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-mask-icon.svg http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-180x180.png http://localhost/wp-content/themes/gds-blogs/build/node_modules/govuk-frontend/dist/govuk/assets/images/govuk-apple-touch-icon-167x167.png
> 
> I'm not sure where Serena got those images from (maybe she generated them herself?) but this might be a good opportunity to bring our markup inline with what the govuk-frontend suggests here: https://design-system.service.gov.uk/styles/page-template/, so that the images referenced are the ones that exist in the govuk-frontend assets.

Ah yes sorry - should have spotted that. I have added the recommended markup, corrected the links and added the manifest.json

RobjS reviewed (approved):
👍
